codespell_lib/data/dictionary.txt

```diff
@@ -15205,6 +15217,7 @@ firmwear->firmware
 firmwqre->firmware
 firmwre->firmware
 firmwware->firmware
+firs->first, furs,
```

Comment:
I really think we should have 'limbo' dictionary that we can throw these words in to that don't fall in to rare or code or shown by default. Since we are parsing source code, this is a valid source code mistake. I'll remove it now but will add it to my separate stash.

Reply:
Maybe name this dictionary as `dictionary_likely.txt` ? It is very likely to be an error for such entry like `firs->first, furs,` where the entry word `firs` is still a valid word. But this dictionary should not enable by default.